website/addons/dryad/model.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from framework.exceptions import HTTPError
+
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+from website.addons.base import StorageAddonBase
+
+from website.addons.dryad import settings as dryad_settings
+#from website.addons.dryad import serializer as serializer
+
+from pyDryad import Dryad
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+
+    def delete(self, **kwargs):
+        super(AddonDryadUserSettings, self).delete()
+
+    @property
+    def has_auth(self):
+        return True
+
+
+class AddonDryadNodeSettings(StorageAddonBase, AddonNodeSettingsBase):
+    """
+        A Dryad node is a collection of packages. Each package is specified by a DOI, and the title is saved automatically
+    """
+    dryad_package_doi = fields.StringField()
+    complete = True
```

Comment:
^ I'm just now realizing `has_auth = True` may be totally accurate in the Dryad use case.